crates/re_time_panel/src/lib.rs

```diff
@@ -765,18 +764,31 @@ impl TimePanel {
         } else {
             // One row:
             let re_ui = &ctx.re_ui;
-            let times_per_timeline = ctx.entity_db.times_per_timeline();
+            let times_per_timeline = entity_db.times_per_timeline();
 
             self.time_control_ui
                 .play_pause_ui(time_ctrl, re_ui, times_per_timeline, ui);
             self.time_control_ui
                 .timeline_selector_ui(time_ctrl, times_per_timeline, ui);
             self.time_control_ui.playback_speed_ui(time_ctrl, ui);
             self.time_control_ui.fps_ui(time_ctrl, ui);
-            current_time_ui(ctx, ui, time_ctrl);
+            current_time_ui(ctx, entity_db, ui, time_ctrl);
 
             ui.with_layout(egui::Layout::right_to_left(egui::Align::Center), |ui| {
                 help_button(ui);
+
+                #[cfg(debug_assertions)]
+                {
+                    if ui
+                        .selectable_label(ctx.app_options.show_blueprint_timeline, "View Blueprint")
+                        .clicked()
```

Comment:
Again, a "Recording"/"Blueprint" pair of radio buttons I think would make this more clear.